pkg/controller/machineset/machineset_controller.go

```diff
@@ -230,12 +245,188 @@ func (c *Controller) processNextWorkItem() bool {
 	}
 	defer c.queue.Done(key)
 
+	// create/delete ansible jobs
 	err := c.syncHandler(key.(string))
 	c.handleErr(err, key)
 
+	// create/delete Machine objects
+	err = c.syncHandler2(key.(string))
+	if err != nil {
+		c.logger.Errorf("JDIAZ error handing machine objects")
+	}
+
 	return true
 }
 
+func (c *Controller) syncMachineSet(key string) error {
+	c.logger.WithField("key", key).Debugf("JDIAZ synching machineset %v", key)
+
+	namespace, name, err := cache.SplitMetaNamespaceKey(key)
+	if err != nil {
+		c.logger.Errorf("JDIAZ error parsing namespace out of machineset name %v", key)
+		return err
+	}
+
+	ms, err := c.machineSetsLister.MachineSets(namespace).Get(name)
+	if errors.IsNotFound(err) {
+		c.logger.WithField("key", key).Errorf("JDIAZ couldn't find machineset %v", key)
+		return err
+	}
+	if err != nil {
+		c.logger.Errorf("JDIAZ problem fetching machineset %v", key)
+		return err
+	}
+	if ms.Spec.NodeType == "Master" {
```

Comment:
This should be ignored in `addMachineSet`, `updateMachineSet`, and `deleteMachineSet`.